File: src/main/java/ru/hollowhorizon/fancychunks/ext/RenderRegionExt.java
package ru.hollowhorizon.fancychunks.ext;

import me.jellysquid.mods.sodium.client.gl.device.CommandList;
import me.jellysquid.mods.sodium.client.render.chunk.RenderSection;

import java.util.List;

public interface RenderRegionExt {
    void updateChunksFade(List<RenderSection> var1, ChunkShaderInterfaceExt var2, CommandList var3);
}

File: src/main/java/ru/hollowhorizon/fancychunks/ext/RenderRegionExt.kt
package ru.hollowhorizon.fancychunks.ext

import me.jellysquid.mods.sodium.client.gl.device.CommandList
import me.jellysquid.mods.sodium.client.render.chunk.RenderSection

interface RenderRegionExt {
    fun updateChunksFade(chunks: MutableList<RenderSection>, shader: ChunkShaderInterfaceExt, commandList: CommandList)
    fun getChunkData(x: Int, y: Int, z: Int): FloatArray
    fun competeChunkFade(x: Int, y: Int, z: Int, completeFade: Boolean)
}